src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpEnvironmentProxy.cs

```diff
@@ -135,8 +138,8 @@ public static bool TryCreate(out IWebProxy proxy)
                 proxy = null;
                 return false;
             }
-
-            proxy = new HttpEnvironmentProxy(httpProxy, httpsProxy, Environment.GetEnvironmentVariable(EnvNoProxyLC));
+			string noProxy = Environment.GetEnvironmentVariable(EnvNoProxyLC) ?? Environment.GetEnvironmentVariable(EnvNoProxyUC);
```

Comment:
Nit: indent should be spaces instead of tabs

Reply:
@stephentoub How did you notice that? :)  I notice it now when I mouse click and drag over that whitespace area. But I was wondering if the GitHub UI has a way to show "control characters" in the whitespace?

Reply:
Mainly just that it wasn't lined up with the rest of the code, and by a large enough margin that generally suggests a tab vs spaces issue; then selecting the whitespace with my mouse confirmed it.

Reply:
When I look at the code in the "Files changed" tab, the alignment problem goes away!  So, that is why I asked the question.
![image](https://user-images.githubusercontent.com/7684835/54079739-000f2c00-4297-11e9-96b5-9ab6d14de719.png)

But if I look at the code in the conversation tab, then I see the alignment difference.
![image](https://user-images.githubusercontent.com/7684835/54079744-1a490a00-4297-11e9-89b2-e5ff934a01df.png)

I wish GitHub UI was consistent! :)

Reply:
Interesting, I see it the same in both places.  Maybe it's to do with the browser? (I'm using Edge.)  Or do you have any extensions that might be messing with the layout?

Reply:
>Maybe it's to do with the browser? (I'm using Edge.) Or do you have any extensions that might be messing with the layout?

I'm using Chrome.

Reply:
I fixed this in last commit